src/main/java/commands/ListCommand.java

```diff
@@ -66,6 +66,7 @@ private String listCards(Access access) {
     }
 
     private String listModules(Access access) {
```

Comment:
if the list command is already separated out, are these two methods still needed?